src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/cas2/ApplicationNoteServiceTest.kt

```diff
@@ -463,6 +463,21 @@ class ApplicationNoteServiceTest {
       }
     }
 
+    @Nested
+    inner class WhenAssessmentIsNotFound {
+      @Test
+      fun `returns Not Found`() {
```

Comment:
ah yep I see the unit tests are here :)
It's a minor bit of feedback, but might help in future when you're organising work/git history to think about what makes an atomic commit - I think the general rule is that each commit should be able to build and run on its own, which means the tests should be in the same commit imo.  (not saying that I'm always perfect at this 😆 )

Reply:
Squashed into one now.